groundstation/static/js/components/LiveCommands.jsx

```diff
@@ -196,7 +196,7 @@ class LiveCommands extends Component {
                 <div>
                     <Paper style={{height:'70%', overflow: 'auto'}}>
                         <Typography className='header-title' variant='h5' style={{padding: '10px', margin: '20px'}}>Live Commands</Typography>
-                        <CommunicationsList autoScroll={true} displayLog={this.state.displayLog} isEmpty={this.state.isEmpty}/>
+                        <CommunicationsList autoScroll={true} displayLog={this.state.displayLog} isEmpty={this.state.isEmpty} showQueueButton={false}/>
```

Comment:
```suggestion
                        <CommunicationsList autoScroll={true} displayLog={this.state.displayLog} isEmpty={this.state.isEmpty} showQueueButton={true}/>
```
Try experimenting with the queue button being visible on the Live Commands page (it adds convenience if an operator wants to immediately dequeue a command they had just send). If it works as intended, we can remove the `showQueueButton` prop and just have it usable anytime there is a communication list.

Reply:
I tried to implement this but the PATCH request I'm trying to make when the queue/de-queue button is clicked is not working from the Live Commands page for some reason and I can't figure out why. It's giving 404